platform/web/api/web_tools_editor_plugin.cpp

```diff
@@ -75,7 +75,7 @@ void WebToolsEditorPlugin::_download_zip() {
 	const String project_name_safe = project_name.to_lower().replace(" ", "_");
```

Comment:
I think this is still incomplete. The comment above suggests that the name is sanitized somehow, but all we do is change it to be in lower case, and replace spaces with underscores. The given example will not be achieved with that. We probably want to call `validate_filename()` in here somewhere.

```suggestion
	const String project_name_safe = project_name.validate_filename().to_lower().replace(" ", "_");
```

But do note that it doesn't seem to replace all the symbols like the example suggests (e.g. it considers single `'` to be safe).

Reply:
cc @Calinou since this was your PR originally https://github.com/godotengine/godot/pull/49598

Reply:
Thank you, I will look into this further. Note: I haven't had a chance to test this because I still need to get a server up to do it, but I wanted to get at least the cause of the error fixed.

Reply:
Slight concern, isn't it the job of `get_safe_dir_name` to do the validating? I would have expected that we'd need to do no further sanitisation than that.

Reply:
You are correct. Despite the name we also use it to sanitize other strings, and it does a bit more than `validate_filename` (though I believe `'` is still allowed). This is good as is then.